File: examples/java/multiple_files/ProgBroken.java
import java.util.Scanner;

public class ProgBroken {
	public static void main(String[] args) {
		switch ((int)(Math.random() * 4)) {
			case 0: // correct 
				Logic.func();
			break;
			case 1: // incorrect
				System.out.println("grapefruit");
			break;
			case 2: // timeouted
				try { Thread.sleep(100 * 1000); } catch (InterruptedException ex) {}
			break;
			case 3: // couldn't execute
				System.err.println("RNGesus decided this should be an error");
				System.exit(17);
			break;
		}
	}
}

File: examples/python/prog_broken.py
import random, time, sys;
r = random.randint(0, 3);
if   r == 0: # correct
	a = int(input());
	b = int(input());
	print(a + b);
	print(a - b);
	print(a * b);
elif r == 1: # incorrect
	print("banana");
elif r == 2: # timeouted
	time.sleep(100);
elif r == 3: # couldn't execute
	print("RNGesus decided this should be an error");
	sys.exit(17);
pass
